traversal/iterator/GraphIterator.java

```diff
@@ -104,9 +105,11 @@ else if (state == State.INIT) {
             return state == State.FETCHED;
         } catch (Throwable e) {
             // note: catching runtime exception until we can gracefully interrupt running queries on tx close
-            if (e instanceof GraknException && ((GraknException) e).code().isPresent()
-                    && ((GraknException) e).code().get().equals(TRANSACTION_CLOSED.code())) {
-                LOG.debug("Transaction was closed during graph iteration");
+            if (e instanceof GraknException && ((GraknException) e).code().isPresent()) {
+                String code = ((GraknException) e).code().get();
+                if (code.equals(TRANSACTION_CLOSED.code()) || code.equals(RESOURCE_CLOSED.code())) {
+                    LOG.debug("Transaction was closed during graph iteration");
+                }
```

Comment:
this helps reduce the frequency at which we print graph procedures in "error" mode, which look like bad exceptions in the logs and are therefore bad UX